src/app/views/river-detail/components/main-tab/components/beta-box/beta-box.vue

```diff
@@ -189,68 +190,72 @@ export default {
       river: state => state.RiverDetail.data,
       editMode: state => state.Global.editMode,
       gages: state => state.RiverGages.data?.gauges ?? [],
-      metrics: state => state.RiverGages.data?.metrics ??[]
+      metrics: state => state.RiverGages.data?.metrics ?? []
     }),
-    releases() {
-      return this.$store.getters['RiverEvents/releases']
+    releases () {
+      // the zero-index in this will be the last or next release.
+      return this.$store.getters['RiverEvents/releasesFromTodayThenPast']
```

Comment:
since beta box just uses a single release -- the next one -- why not fully abstract the reliance on the ordering of an array from another piece of code out of this and just write a getter called `RiverEvents/nextRelease`?

it would also simplify the template code -- `v-if="nextRelease"` etc. etc.